### PR TITLE
Nette 3.0

### PR DESCRIPTION
**Pull request by miranovy**
Added support to Nette/database 3.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,12 +18,16 @@
   ],
   "require": {
     "php": ">= 7.1",
```
Comment (f3l1x) [inline composer.json]: ^7.2

File: composer.json [hunk 2]
```diff
@@ -18,12 +18,16 @@
...
-    "nette/database": "~2.4.7"
+    "nette/database": "^3.0.2"
```
Comment (f3l1x) [inline composer.json]: ~3.0.2

**Comment by f3l1x** [comment]
Hi. Thanks a lot. Could please do a last step and move composer scripts to makefile? 

Take a look at here https://github.com/contributte/neonizer/commit/ea7508977ccba2dc352680ab1af5da3214f5c405.

**Comment by miranovy** [comment]
Makefile added :)

**Comment by f3l1x** [comment]
Awesome. Thanks.

**Pull request merged**